test/render.js

```diff
@@ -778,4 +778,34 @@ describe('render', () => {
 		);
 		expect(res).to.equal('<select><option selected value="2">2</option></select>');
 	});
+
+	describe('suspense', () => {
+		it('should render suspension', () => {
+			function LazyComp() { return <p>Hello Lazy!</p>; }
+			const Lazied = lazy(() => Promise.resolve({ default: LazyComp }));
+
+			const toBeRendered = (
+				<section>
+					<Suspense fallback={<div>Fallback...</div>}>
+						<article>
+							<Lazied />
+						</article>
+					</Suspense>
+				</section>
+			);
+
+			renderAsync(toBeRendered)
```

Comment:
The `Promise` needs to be returned, otherwise the test will be marked as complete before the promise is resolved

Reply:
Good catch. Thx!